src/app/components/Card/Card.module.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  margin: 0 auto;
```

Comment:
I would prefer not setting a margin here but rather the parent centering the element